Adapters/Vungle/ISVungleAdapter/ISVungleBannerDelegate.m

```diff
@@ -11,29 +11,24 @@
 @implementation ISVungleBannerDelegate
 
 - (instancetype)initWithPlacementId:(NSString *)placementId
-                      containerView:(UIView *)containerView
                         andDelegate:(id<ISBannerAdapterDelegate>)delegate {
     self = [super init];
     if (self) {
         _placementId = placementId;
-        _containerView = containerView;
         _delegate = delegate;
     }
     return self;
 }
 
-#pragma mark - Banner Delegate
+#pragma mark - VungleBannerView Delegate
 
-- (void)bannerAdDidLoad:(VungleBanner * _Nonnull)banner {
+- (void)bannerAdDidLoad:(VungleBannerView * _Nonnull)bannerView {
     LogAdapterDelegate_Internal(@"placementId = %@", self.placementId);
-
-    [self.delegate adapterBannerDidLoad:self.containerView];
-    dispatch_async(dispatch_get_main_queue(), ^{
-        [banner presentOn:self.containerView];
-    });
+    [self.delegate adapterBannerDidLoad:bannerView];
+    // aki:todo - might need to retun the w and h to IS side...?
```

Comment:
Not sure if iOS have similar API like Android: `onBannerAdLoaded(view, layoutParams)`

Reply:
`- (void)adapterBannerDidLoad:(UIView *)bannerView;` 
There is no argument "layoutParams" in adDidLoad callback in iOS. 
I will verify if it works as expected without this param.

Reply:
There is no similar delegate callback API which can send "layout param" in iOS side.